src/components/block/api.ts

```diff
@@ -0,0 +1,152 @@
+import Block from './index';
+import {BlockToolData, ToolConfig} from '../../../types/tools';
+import {SavedData} from '../../types-internal/block-data';
+import {BlockAPI as BlockAPIInterface} from '../../../types/api';
+
+function BlockAPI(block: Block) {
+  const blockAPI: BlockAPIInterface =  {
+    /**
+     * Tool name
+     *
+     * @return {string}
+     */
+    get name(): string {
+      return block.name;
+    },
+
+    /**
+     * Tool config passed on Editor's initialization
+     *
+     * @return {ToolConfig}
+     */
+    get settings(): ToolConfig {
```

Comment:
I think it is not required by outside

Reply:
WIll rename it to `config` as some BlockTunes might rely on Tool's configuration